src/main/groovy/org/mockito/release/notes/format/DetailedFormatter.java

```diff
@@ -53,13 +53,20 @@ public String formatReleaseNotes(Collection<ReleaseNotesData> data) {
         return sb.toString().trim();
     }
 
-    static String releaseSummary(Date date, ContributionSet contributions, Map<String, Contributor> contributors,
+    static String releaseSummary(String version, Date date, ContributionSet contributions, Map<String, Contributor> contributors,
                                  String vcsCommitsLink, String publicationRepository) {
         return authorsSummary(contributions, contributors, vcsCommitsLink) +
-                " - *" + DateFormat.formatDate(date) + "*" + " - published to " + publicationRepository + "\n" +
+                " - *" + DateFormat.formatDate(date) + "*" + " - published to " + getBintrayBadge(version, publicationRepository) + "\n" +
                 authorsSummaryAppendix(contributions, contributors);
     }
 
+    private static String getBintrayBadge(String version, String publicationRepository) {
+        StringBuilder sb = new StringBuilder();
```

Comment:
No need to use StringBuilder here, regular string concatenation would be more readable. Also 
sb.append("[![Bintray]").append("(https://img.shields.io/badge/Bintray-") can be replaced with one literal.